modules/combined/doc/content/newsletter/2018_08_17.md

```diff
@@ -0,0 +1,95 @@
+# MOOSE Newsletter (August 2018)
+
+## MooseDocs Plots
+
+It is now possible to include line plots in MOOSE markdown. The plots are created using
+[plotly]; additional information may be found on the [plotly.md] page.
+
+```markdown
+!plot scatter data=[{'x':[1,2,3,4], 'y':[2,4,6,8], 'name':'2x'},
+                    {'x':[1,2,3,4], 'y':[1,4,9,16], 'name':'x*x', 'marker':{'size':14}}]
+```
+
+!plot scatter data=[{'x':[1,2,3,4], 'y':[2,4,6,8], 'name':'2x'},
+                    {'x':[1,2,3,4], 'y':[1,4,9,16], 'name':'x*x', 'marker':{'size':14}}]
+
+## MeshModifier for creating lower dimensional blocks
+
+It is now possible to generate a lower dimensional block from sidesets using the
+MOOSE MeshModifier `LowerDBlockFromSideset`. The required parameters to use the
+modifier are `sidesets` which is a vector of `BoundaryID`s from which to build
+the new block and `new_block_id` which is the `SubdomainID` of the new block. An
+optional parameter is `new_block_name` if the user wants to define a name for
+the newly created block. The order of the lower dimensional elements created
+will match the order of the parent element. E.g. if we are creating a block from
+`QUAD9` or `TRI6` faces, then the newly created 1D block will have elements of type
+`EDGE3`. `QUAD4` and `TRI3` elements would yield `EDGE2`. 3D behavior is
+analogous. The snippet below shows input file syntax for using
+`LowerDBlocksFromSideset`:
+
+```
+[MeshModifiers]
+  [./lower]
+    type = LowerDBlockFromSideset
+    sidesets = '0 1 2 3'
+    new_block_id = 10
+    new_block_name = 'lower_d'
+  [../]
+[]
+```
+
+
+## Threaded General User Objects
+
+Up until now, general user objects were executed in a single threaded region, even if the
+application ran with multiple-threads.  Now, it is possible to have threaded general user objects.
+Users need to inherit from `ThreadedGeneralUserObject` class and MOOSE will create one threaded
+instance per thread. Then, it is possible to safely access the user object from each threaded. This
```

Comment:
typo: "safely access the user object from each threaded." 

Should this be "each thread" or "threaded instance" or ?